react/src/components/atoms/forms/InputRadio/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './style.css';
+
+class InputRadio extends React.Component {
+  constructor(props) {
+    super(props);
+    this.handleChange = this.handleChange.bind(this);
+  }
+
+  handleChange(event) {
+    const selected = event.target.value;
+    const value = this.props.value;
+    if (typeof this.props.onChange === 'function') {
+      this.props.onChange(selected, value, event);
+    }
+  }
+
+  render() {
+    const radioClasses = classNames({
+      'ma__input-radio': !this.props.outline,
+      'ma__input-radio--outline': this.props.outline
+    });
+    const inputClasses = classNames({
+      'ma__input-radio__control': !this.props.outline,
+      'ma__input-radio--outline__control': this.props.outline,
+      'ma__input-radio--outline__control--error': this.props.error
+    });
+    const labelClasses = classNames({
+      'ma__input-radio__label': !this.props.outline,
+      'ma__input-radio--outline__label': this.props.outline,
+      'ma__input-radio--outline__label--error': this.props.error
+    });
+    return(
+      <div className={radioClasses}>
+        <input
+          checked={this.props.checked}
+          name={this.props.name}
+          type="radio"
+          value={this.props.value}
+          id={this.props.id}
+          required={this.props.required}
+          onChange={this.handleChange}
```

Comment:
I'm thinking this should actually just be `this.props.onChange`. This component doesn't do anything with the onChange event at all except pass it and props.value. The component that's actually using InputRadio already has the value set to `props.value`, so there's no reason to pass it to the function.
If this is changed to `this.props.onChange`, then you can remove handleChange entirely and change this component to be functional instead of a class.